pkg/commands/process/settings/rules/ruby/lang/weak_encryption/testdata/openssl_dsa.rb

```diff
@@ -1,5 +1,11 @@
+size = 2048
 cipher = OpenSSL::Cipher.new('aes-128-cbc')
-dsa_encrypt = OpenSSL::PKey::DSA.new(2048)
+rsa_encrypt = OpenSSL::PKey::RSA.new(size)
```

Comment:
We'd want these changes in the RSA file: https://github.com/Bearer/bearer/blob/main/pkg/commands/process/settings/rules/ruby/lang/weak_encryption/testdata/openssl_rsa.rb

Reply:
nice catch